wsgi_intercept/__init__.py

```diff
@@ -640,7 +658,8 @@ def connect(self):
                             else:
                                 self._check_hostname = self.check_hostname
                 except (ImportError, AttributeError):
-                    pass
+                    import traceback
+                    traceback.print_exc()
```

Comment:
Unsure if you want me to remove this (was used for debugging) or leave it there?

Reply:
It's fine to leave it there.